config/initializers/constants.rb

```diff
@@ -205,6 +205,21 @@ class Constants
 
   ACADEMY_BL_LINK = 'https://scinote.net/academy/?utm_source=SciNote%20software%20BL&utm_medium=SciNote%20software%20BL'.freeze
 
+  TWO_FACTOR_URL = {
+    google: {
+      android: 'https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2',
+      ios: 'https://apps.apple.com/us/app/google-authenticator/id388497605'
```

Comment:
Is it account specific URL?

Reply:
no, it's the id of the app

Reply:
https://apps.apple.com/us/app/alex-is-the-king/id388497605